messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/jms/ExternalPooledConnectionFactoryService.java

```diff
@@ -210,15 +210,6 @@ public ExternalPooledConnectionFactoryService(String name, TransportConfiguratio
         this.enlistmentTrace = enlistmentTrace;
     }
 
-    private void initJNDIBindings(List<String> jndiNames) {
-        // create the definition with the 1st jndi names and create jndi aliases for the rest
-        String jndiName = jndiNames.get(0);
-        this.bindInfo = ContextNames.bindInfoFor(jndiName);
-        this.jndiAliases = new ArrayList<>();
```

Comment:
I think this PR leaves this.jndiAliases and createJNDIAliases around as cruft.

Reply:
this has been addressed